src/components/bulk-email-tool/bulk-email-form/BulkEmailForm.jsx

```diff
@@ -115,127 +153,102 @@ export default function BulkEmailForm(props) {
           <Form.Control name="subject" className="w-lg-50" onChange={onSubjectChange} />
           {!emailFormValidation.subject && (
             <Form.Control.Feedback className="px-3" hasIcon type="invalid">
-              <FormattedMessage
-                id="bulk.email.form.subject.error"
-                defaultMessage="A subject is required"
-                description="An Error message located under the subject line. Visible only on failure."
-              />
+              {intl.formatMessage(messages.bulkEmailFormSubjectError)}
             </Form.Control.Feedback>
           )}
         </Form.Group>
         <Form.Group controlId="emailBody">
-          <Form.Label>
-            <FormattedMessage
-              id="bulk.email.body.label"
-              defaultMessage="Body:"
-              description="Email Body label. Meant to have colon or equivilant punctuation."
-            />
-          </Form.Label>
+          <Form.Label>{intl.formatMessage(messages.bulkEmailBodyLabel)}</Form.Label>
           <TextEditor onInit={onInit} />
           {!emailFormValidation.body && (
             <Form.Control.Feedback className="px-3" hasIcon type="invalid">
-              <FormattedMessage
-                id="bulk.email.form.body.error"
-                defaultMessage="The message cannot be blank"
-                description="An error message located under the body editor. Visible only on failure."
-              />
+              {intl.formatMessage(messages.bulkEmailFormBodyError)}
             </Form.Control.Feedback>
           )}
         </Form.Group>
         <div>
+          <p>{intl.formatMessage(messages.bulkEmailInstructionsProofreading)}</p>
           <p>
-            <FormattedMessage
-              id="bulk.email.instructions.proofreading"
-              defaultMessage="We recommend sending learners no more than one email message per week. Before you send your email, review
-              the text carefully and send it to yourself first, so that you can preview the formatting and make sure
-              embedded images and links work correctly."
-              description="A set of instructions to give users a heads up about the formatting of the email they are about to send"
-            />
-          </p>
-          <p>
-            <strong>
-              <FormattedMessage id="bulk.email.instructions.caution" defaultMessage="Caution!" />
-            </strong>
-            <FormattedMessage
-              id="bulk.email.instructions.caution.message"
-              defaultMessage=" When you select Send Email, your email message is added to the queue for sending,
-              and cannot be cancelled."
-              description="A warning about how emails are sent out to users"
-            />
+            <strong>{intl.formatMessage(messages.bulkEmailInstructionsCaution)}</strong>
+            {intl.formatMessage(messages.bulkEmailInstructionsCautionMessage)}
```

Comment:
What should we do with this messaging if the email author wants to schedule their message? The context doesn't make sense because eventually we _will_ allow a message to be cancelled. 

Should this only display when scheduling is disabled or the `Schedule this email...` checkbox isn't selected?